cabal-plan.cabal

```diff
@@ -55,6 +55,7 @@ executable cabal-plan
   -- dependencies which require version bounds
   build-depends: mtl            >= 2.2.1 && < 2.3
                , ansi-terminal  >= 0.6.2 && < 0.7
+               , optparse-applicative >= 0.13.0 && < 0.14
```

Comment:
From your comment, I gather you only have evidence by testing this with `optparse-applicative-0.13.2.0`, so the proper inferred version range would be `>= 0.13.2.0 && < 0.14`

However, I just tested this w/  `optparse-applicative-0.13.0.0` and it does indeed work, so there's no need to change anything here either!